sdv/constraints/tabular.py

```diff
@@ -803,9 +777,12 @@ def __init__(self, column_name, strict_boundaries=False):
 class Range(Constraint):
     """Ensure that the ``middle_column_name`` is between ``low`` and ``high`` columns.
 
-    The transformation strategy works by replacing the ``middle_column_name`` with a
-    scaled version and then applying a logit function. The reverse transform
-    applies a sigmoid to the data and then scales it back to the original space.
+    The transformation strategy works as the Inequality constraint but with two
```

Comment:
minor: change to "works the same as the Inequality..."

Reply:
Done in [18bdaf0](https://github.com/sdv-dev/SDV/pull/1404/commits/18bdaf09a65211edb2e8c54ccffd057a72fa68d4)